src/SFA.DAS.EmployerAccounts/Data/EmployerAccountsDbContext.cs

```diff
@@ -74,6 +75,7 @@ protected override void OnModelCreating(DbModelBuilder modelBuilder)
             modelBuilder.Entity<UserAccountSetting>().HasRequired(u => u.Account);
             modelBuilder.Entity<UserAccountSetting>().HasRequired(u => u.User);
             modelBuilder.Entity<UserAccountSetting>().ToTable("UserAccountSettings");
+            modelBuilder.Entity<RunOnceJob>().HasKey(j => j.Name);
```

Comment:
What do you think to moving the table to the `dbo` schema (similar to the other infrastructure related tables e.g. `HealthChecks`, `OutboxData`, etc)? i.e. `.ToTable("RunOnceJobs", "dbo");`